ui/components/connections/MeshsyncTable.js

```diff
@@ -46,18 +51,18 @@ export default function MeshSyncTable({ classes, updateProgress, search }) {
       name: 'metadata.name',
       label: 'Name',
       options: {
-        sort: true,
+        sort: false,
         sortThirdClickReset: true,
-        customHeadRender: function CustomHead({ index, ...column }, sortColumn, columnMeta) {
-          return (
-            <SortableTableCell
-              index={index}
-              columnData={column}
-              columnMeta={columnMeta}
-              onSort={() => sortColumn(index)}
-            />
-          );
-        },
+        // customHeadRender: function CustomHead({ index, ...column }, sortColumn, columnMeta) {
```

Comment:
Should be removed if not used

Reply:
removed for now, so if needed can be added again